src/administration/integrations/gitlab.md

```diff
@@ -49,7 +49,7 @@ The previous command, if succesful should output the configuration of the integr
 | hook_url | https://{region}.platform.sh/api/projects/{projectid}/integrations/{hook_id}/hook |
 ```
 
-Now, copy the hook URL.
+If you don't see the CLI outputs `Webhook created successfully`, you need to add the webhook manually. Copy the hook URL and continue the below.
```

Comment:
"continue the below" is clumsy grammar.  What's the change here exactly?

Reply:
The change is that the CLI will automatically ensure webhooks, if it can (i.e. if the token is correct and has enough permissions).

Suggestion:

> If you see the message "Failed to read or write webhooks", you will need to add a webhook manually:
> 
> - Copy the hook URL shown in the message.
> - Go to your GitLab repository and click `Settings > Integrations`.
> - Paste the hook URL. In the Triggers section choose `Push events`, `Tag push events` and `Merge Request events`. Click on `Add webhook`.